docs/source/conf.py

```diff
@@ -85,6 +85,23 @@ def __getattr__(self, name):
     def __setattr__(self, name, value):
         return setattr(sys.stdout, name, value)
 pwnlib.context.ContextType.defaults['log_console'] = stdout()
+
+offline=True
+binutils_mips=False
+binutils_arm=False
+binutils_amd64=False
+binutils_i386=False
+binutils_aarch64=False
+binutils_powerpc=False
+binutils_thumb=False
+qemu_mips=False
+qemu_arm=False
+qemu_amd64=False
+qemu_i386=False
+qemu_aarch64=False
+qemu_powerpc=False
+qemu_thumb=False
+travis=False
```

Comment:
I don't understand the intent of `not travis`.  The only supported environments for running the Pwntools tests are:

1.) From the included Docker image, which sets up the environment similar to Travis.  See `travis/Docker` or just run it: `make -C travis/docker ANDROID=no`.
2.) On Travis itself.

Reply:
It is poorly named. It is used when certain binaries are expected to be in certain locations. It should be named `not fhs` or something similar instead.

Reply:
When will e.g. `/bin/sh` not exist at `/bin/sh`?